packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -147,11 +148,17 @@ export default class CleanWebpackPlugin {
    * Warning: It is recommended to initially clean your build directory outside
    * of webpack to minimize unexpected behavior.
    */
-  handleInitial(): void {
+  async handleInitial(): Promise<void> {
```

Comment:
You _cannot_ use `async` here. This creates a floating promise when you call `this.handleInitial` later in `apply()`.

Reply:
I see, thank you for the suggestion I will revert back to sync